sql/firefox_accounts_derived/fxa_log_content_other_events_v1/query.sql

```diff
@@ -0,0 +1,71 @@
+SELECT
+  jsonPayload.event,
+  jsonPayload.flow_time,
+  jsonPayload.locale,
+  jsonPayload.useragent,
+  jsonPayload.country,
+  jsonPayload.entrypoint,
+  jsonPayload.flow_id,
+  jsonPayload.region,
+  jsonPayload.service,
+  jsonPayload.utm_campaign,
+  jsonPayload.utm_content,
+  jsonPayload.utm_medium,
+  jsonPayload.utm_source,
+  -- timestamp, -- This is a static value in 1970 for all records
```

Comment:
Does it seem reasonable to leave off `timestamp` completely, considering it doesn't seem to be meaningful?

Reply:
Sounds good to me.